hllplus/hllplus.go

```diff
@@ -81,6 +81,11 @@ func (s *HLL) Merge(other *HLL) {
 		return
 	}
 
+	// Make sure receiver is allocated.
+	if len(s.normal) == 0 {
```

Comment:
sorry, I noticed this on Fri and forgot. can you pls check for all "accesses" to `s.normal` in the file and make sure we have `if len(s.normal) == 0 {` above. maybe move to `s.ensureNormal()` helper to avoid repetition